changelogs/fragments/58370-mysqldb-allow-multiple_databases_create_delete_operation.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- mysql_db now supports creation and deletion of multiple databases (https://github.com/ansible/ansible/issues/58370)
```

Comment:
I would add another line that it not behaves better w.r.t. `changed` result in `check_mode`.

Reply:
done